src/nprintml/util/interface.py

```diff
@@ -1,89 +1,6 @@
 import argparse
```

Comment:
absolutely nothing changed here … just code moving around (`util.py` → `util/` and stuff from `net.step`).